.github/workflows/cd.yml

```diff
@@ -0,0 +1,15 @@
+name: cd workflow
+
+on:
+  push:
+    branches:
+      - master
+    
+jobs:
+ deploy:
+   name: Deploy
```

Comment:
needs: other steps

Reply:
united workflows into one with the condition whether there is a push to the master